uwg_schema/ref_bld_template.py

```diff
@@ -225,29 +228,32 @@ class BEMDef(NoExtraBaseModel):
     """Building Energy Model (BEM) definition."""
     type: constr(regex='^BEMDef$') = 'BEMDef'
 
-    bldtype: int = Field(
+    bldtype: str = Field(
         ...,
-        ge=0,
-        description='Number between 0 and 15 corresponding to the following '
-        'building types: FullServiceRestaurant (0), Hospital (1), LargeHotel (2), '
-        'LargeOffice (3), MediumOffice (4), MidRiseApartment (5), OutPatient (6), '
-        'PrimarySchool (7), QuickServiceRestaurant (8), SecondarySchool (9), '
-        'SmallHotel (10), SmallOffice (11), StandaloneRetail (12), StripMall (13), '
-        'SuperMarket (14), Warehouse (15). Additional building types can be defined '
-        'with a number greater then 15. This value is used to reference the fraction '
-        'of urban area the BEMDef object defines in the UWG bld matrix.'
+        description='Text referring to a building type. To reference (or overwrite) '
+        'a DOE reference building, text must be one of the following: '
+        '"fullservicerestaurant", "hospital", "largehotel", "largeoffice", '
+        '"mediumoffice", "midriseapartment", "outpatient", "primaryschool", '
+        '"quickservicerestaurant", "secondaryschool", "smallhotel", "smalloffice", '
+        '"standaloneretail", "stripmall", "supermarket", or "warehouse". This value '
+        'along with the builtera is used to reference the fraction of urban area the '
+        'building defines in the UWG bld matrix.'
     )
 
-    builtera: int = Field(
+    builtera: str = Field(
         ...,
-        ge=0,
-        le=2,
-        description='Number between 0 and 2 corresponding to the following '
-        'built eras: Pre-1980s (0), Post1980s (1), New construction (2). '
-        'This value is used to reference the fraction of urban area the BEMDef '
-        'object defines in the UWG bld matrix.'
+        description='Text defining building built era. Must be one of the following:'
+        '"pre80" (pre-1980s), "pst80" (post-1980s), or "new" (new constrution).'
+        'This value and the bldtype is used to reference the fraction of urban area'
+        'the building defines in the UWG bld matrix.'
```

Comment:
Its probably worth emphasizing that this value MUST match what is in the bld matrix.